lib/assets/data_transform_cell/main.js

```diff
@@ -124,8 +124,8 @@ export async function init(ctx, payload) {
         v-bind="$attrs"
         v-bind:class="inputClass"
       >
-      <div class="validation-wrapper">
-        <span class="tooltip right validation-message" :data-tooltip="message" v-if="message">
+      <div class="validation-wrapper" v-if="message">
+        <span class="tooltip right validation-message" :data-tooltip="message">
```

Comment:
@cristineguadelupe I have no idea if this change is OK. Can you please confirm?

Reply:
Yes, it is!